### DISCUSSION TITLE
Plugins support

### DISCUSSION THREAD

**Discussion by jmigual** [JanDeDobbeleer/oh-my-posh#760] [Troubleshoot]
Hello everyone,

I really love oh-my-posh and its configuration syntax. I already use it for PowerShell but I'd like to use it with zsh. The only thing keeping me from doing that is the lack of proper plugin support like oh-my-zsh does where I can place everything in the `plugins` folder and it reads them automatically. Am I missing something and is it possible or would this be a nice addition to have?

I am currently using the following plugins with zsh:
- [zsh-syntax-highlighting](https://github.com/zsh-users/zsh-syntax-highlighting)
- [zsh-autosuggestions](https://github.com/zsh-users/zsh-autosuggestions)

Thanks in advance!

**Comment by JanDeDobbeleer** [comment]
That's actually not possible, although it's perfectly possible to combine oh my zsh and oh my posh at the same time. The only thing oh my posh does is override the prompt, everything else will work in oh my zsh.
